app/assets/stylesheets/shareable.scss

```diff
@@ -0,0 +1,10 @@
+nav.share {display:inline-block;} /* shareable's nav tag */
+nav.share div {float:left;}
+nav.share iframe {border:0; margin-top:0; padding-top:0;} /* if using medium sized buttons */
```

Comment:
Use `//` comments everywhere<br>Merge rule `nav.share iframe` with rule on line 1<br>Avoid qualifying class selectors with an element.<br>Colon after property should be followed by one space